addok_csv/__init__.py

```diff
@@ -23,6 +23,16 @@ def register_http_endpoint(api):
 
 def preconfigure(config):
     config.CSV_ENCODING = 'utf-8-sig'
+    config.CSV_EXTRA_FIELDS = []
+
+
+@config.on_load
+def on_load():
+    if not config.CSV_EXTRA_FIELDS:
```

Comment:
About the name: `FIELDS` is a list of dict, while `CSV_EXTRA_FIELDS` is a list of names.
I thought about switching to something like `CSV_EXTRA_COLUMNS`, but the thing is that we don't want column names, we want actuel fields names, i.e. the column name for the field `extra` will be `result_extra`, and we just want `extra` here.
So I finally keep `CSV_EXTRA_FIELDS`.

Reply:
What about the more general `CSV_EXTRAS`? Even more confusing? 🙃